crates/store/re_protos/proto/rerun/v1alpha1/manifest_registry.proto

```diff
@@ -385,9 +385,13 @@ message Query {
 // This has the exact same semantics as the query of the same name on our `ChunkStore`.
 message QueryLatestAt {
   // Which index column should we perform the query on? E.g. `log_time`.
-  rerun.common.v1alpha1.IndexColumnSelector index = 1;
+  //
+  // Leave this empty to query for static data.
+  optional rerun.common.v1alpha1.IndexColumnSelector index = 1;
 
   // What index value are we looking for?
+  //
+  // Leave this empty to query for static data.
   optional int64 at = 2;
```

Comment:
Related to my previous comment, I'm low-key surprised that this is already optional.